### DISCUSSION TITLE
[Classic/SOD] - Adding more click casting buttons

### DISCUSSION THREAD

**Discussion by Hessutar** [siweia/NDui#267] [Q&A]
Hello! 

Is there a way to add more click casting buttons than the pre configured ones + modifiers? I am using a Razer Naga mouse with 12 side buttons and would like to bind them for click casting.

**Comment by siweia** [comment]
If you bind your extra mouse buttons into default keybind, what is the key string on the default panel?

**Reply by Hessutar** [reply]
I have the extra buttons bound to different keyboard binds and combinations in the Razer software as follows.

![image](https://github.com/siweia/NDui/assets/5693510/041794c7-a295-43ab-a5fd-71d75d6aefc9)
